bitnami/harbor/values.yaml

```diff
@@ -805,6 +807,14 @@ core:
 ##
 jobservice:
   replicas: 1
+
+  ## Update strategy - only really applicable for deployments with RWO PVs attached
+  ## If replicas = 1, an update can get "stuck", as the previous pod remains attached to the 
+  ## PV, and the "incoming" pod can never start. Changing the strategy to "Recreate" will 
+  ## terminate the single previous pod, so that the new, incoming pod can attach to the PV
+  updateStrategy:
```

Comment:
Add a line after the comments:
```diff
## Update strategy - only really applicable for deployments with RWO PVs attached
## If replicas = 1, an update can get "stuck", as the previous pod remains attached to the 
## PV, and the "incoming" pod can never start. Changing the strategy to "Recreate" will 
## terminate the single previous pod, so that the new, incoming pod can attach to the PV
+ ##
```